examples/model_selection/plot_det.py

```diff
@@ -44,58 +44,43 @@
 
     - This example is loosely based on
       :ref:`sphx_glr_auto_examples_classification_plot_classifier_comparison.py`
-      .
+      example.
 
 """
 import matplotlib.pyplot as plt
 
-from sklearn.model_selection import train_test_split
-from sklearn.preprocessing import StandardScaler
 from sklearn.datasets import make_classification
-from sklearn.svm import SVC
 from sklearn.ensemble import RandomForestClassifier
 from sklearn.metrics import detection_error_tradeoff_curve
-from sklearn.metrics import roc_curve
+from sklearn.metrics import plot_roc_curve
+from sklearn.model_selection import train_test_split
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
+from sklearn.svm import LinearSVC
 
 from scipy.stats import norm
-from matplotlib.ticker import FuncFormatter
 
 N_SAMPLES = 1000
 
-names = [
-    "Linear SVM",
-    "Random Forest",
-]
-
-classifiers = [
-    SVC(kernel="linear", C=0.025),
-    RandomForestClassifier(max_depth=5, n_estimators=10, max_features=1),
-]
+classifiers = {
+    "Linear SVM": make_pipeline(StandardScaler(), LinearSVC(C=0.025)),
+    "Random Forest": RandomForestClassifier(
+        max_depth=5, n_estimators=10, max_features=1
+    ),
+}
 
 X, y = make_classification(
     n_samples=N_SAMPLES, n_features=2, n_redundant=0, n_informative=2,
     random_state=1, n_clusters_per_class=1)
 
-# preprocess dataset, split into training and test part
-X = StandardScaler().fit_transform(X)
-
 X_train, X_test, y_train, y_test = train_test_split(
     X, y, test_size=.4, random_state=0)
 
 # prepare plots
-fig, [ax_roc, ax_det] = plt.subplots(1, 2, figsize=(10, 5))
+fig, [ax_roc, ax_det] = plt.subplots(1, 2, figsize=(11, 5))
 
 # first prepare the ROC curve
-ax_roc.set_title('Receiver Operating Characteristic (ROC) curves')
```

Comment:
Shouldn't we put back this title? RIght now the right plot has a title but the left one doesn't

Reply:
Oh good catch.